config/routes.rb

```diff
@@ -24,6 +24,7 @@
 
   resources :subscriptions, only: %i[new create edit update] do
     get :unsubscribe, on: :member
+    resources :feedbacks, only: %i[new update edit], controller: 'job_alert_feedbacks'
```

Comment:
we already have
`resource :feedback, controller: 'general_feedback', only: %i[new create]`

I would call this resource like:
`resources : job_alert_feedbacks, only: %i[new update edit]`